pkg/eve/applications.go

```diff
@@ -28,11 +30,16 @@ type AppInstState struct {
 	ExternalIP   string
 	InternalPort string
 	ExternalPort string
-	Memory       string
-	macs         []string
-	volumes      map[string]uint32
-	deleted      bool
-	infoTime     time.Time
+	MemoryUsed   uint32
+	MemoryAvail  uint32
+	CpuUsage     int
```

Comment:
Some Yetus complaints

Reply:
Modified, thank you

Reply:
Now yetus is finally happy (with several nolint directives unfortunately)